### PR TITLE
misc parameter and method names

### PR DESCRIPTION
**Pull request by jaskarth**
A lot more work on fixing unmapped parameter names and a couple of functions

### CODE REVIEW

File: mappings/net/minecraft/block/sapling/SaplingGenerator.mapping [hunk 1]
```diff
@@ -1,3 +1,12 @@
 CLASS net/minecraft/class_2647 net/minecraft/block/sapling/SaplingGenerator
 	METHOD method_11430 createTreeFeature (Ljava/util/Random;Z)Lnet/minecraft/class_2975;
+		ARG 2 nearFlowers
 	METHOD method_11431 generate (Lnet/minecraft/class_1936;Lnet/minecraft/class_2794;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Ljava/util/Random;)Z
+		ARG 1 world
+		ARG 2 chunkGenerator
+		ARG 3 pos
+		ARG 4 stte
```
Comment (liach) [inline mappings/net/minecraft/block/sapling/SaplingGenerator.mapping]:
```suggestion
		ARG 4 state
```

File: mappings/net/minecraft/world/ChunkRegion.mapping [hunk 2]
```diff
@@ -14,9 +14,15 @@ CLASS net/minecraft/class_3233 net/minecraft/world/ChunkRegion
 	FIELD field_14098 chunks Ljava/util/List;
 	FIELD field_14099 blockTickScheduler Lnet/minecraft/class_1951;
 	FIELD field_20668 biomeAccess Lnet/minecraft/class_4543;
-	METHOD <init> (Lnet/minecraft/class_3218;Ljava/util/List;)V
+	METHOD <init> <init> (Lnet/minecraft/class_3218;Ljava/util/List;)V
+		COMMENT Used during world generation to access multiple chunks in a square area.
+		COMMENT Accessing chunks outside of the area will result in a crash.
```
Comment (liach) [inline mappings/net/minecraft/world/ChunkRegion.mapping:19]: Is this intended to be class docs

File: mappings/net/minecraft/world/MobSpawnerLogic.mapping [hunk 3]
```diff
@@ -11,15 +11,19 @@ CLASS net/minecraft/class_1917 net/minecraft/world/MobSpawnerLogic
 	FIELD field_9158 requiredPlayerRange I
 	FIELD field_9160 maxNearbyEntities I
 	METHOD method_18086 spawnEntity (Lnet/minecraft/class_1297;)V
+		ARG 1 entity
 	METHOD method_8271 getWorld ()Lnet/minecraft/class_1937;
 	METHOD method_8272 serialize (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2487;
```
Comment (liach) [inline mappings/net/minecraft/world/MobSpawnerLogic.mapping]:
```suggestion
	METHOD method_8272 toTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2487;
```
Should just do a refactor here

File: mappings/net/minecraft/world/MobSpawnerLogic.mapping [hunk 4]
```diff
@@ -11,15 +11,19 @@ CLASS net/minecraft/class_1917 net/minecraft/world/MobSpawnerLogic
...
+		ARG 1 tag
 	METHOD method_8273 sendStatus (I)V
 		ARG 1 status
 	METHOD method_8274 setEntityId (Lnet/minecraft/class_1299;)V
+		ARG 1 type
 	METHOD method_8276 getPos ()Lnet/minecraft/class_2338;
 	METHOD method_8277 setSpawnEntry (Lnet/minecraft/class_1952;)V
 		ARG 1 spawnEntry
 	METHOD method_8280 deserialize (Lnet/minecraft/class_2487;)V
```
Comment (liach) [inline mappings/net/minecraft/world/MobSpawnerLogic.mapping]:
```suggestion
	METHOD method_8280 fromTag (Lnet/minecraft/class_2487;)V
```

File: mappings/net/minecraft/world/SpawnHelper.mapping [hunk 5]
```diff
@@ -1,29 +1,54 @@
 CLASS net/minecraft/class_1948 net/minecraft/world/SpawnHelper
 	FIELD field_9292 LOGGER Lorg/apache/logging/log4j/Logger;
+	METHOD method_24930 spawn (Lnet/minecraft/class_1311;Lnet/minecraft/class_3218;Lnet/minecraft/class_2791;Lnet/minecraft/class_2338;)V
+		ARG 0 catagory
+		ARG 1 world
+		ARG 2 chunk
+		ARG 3 pos
+	METHOD method_24931 trySpawnMob (Lnet/minecraft/class_3218;Lnet/minecraft/class_1299;)Lnet/minecraft/class_1308;
```
Comment (liach) [inline mappings/net/minecraft/world/SpawnHelper.mapping]:
```suggestion
	METHOD method_24931 createEntity (Lnet/minecraft/class_3218;Lnet/minecraft/class_1299;)Lnet/minecraft/class_1308;
		ARG 0 world
		ARG 1 type
```
This takes a world and an entity type and returns an entity.

File: mappings/net/minecraft/world/SpawnHelper.mapping [hunk 6]
```diff
@@ -1,29 +1,54 @@
...
+	METHOD method_24932 (Lnet/minecraft/class_3218;Lnet/minecraft/class_1308;D)Z
+		ARG 0 world
+		ARG 1 entity
+	METHOD method_24933 canSpawnMob (Lnet/minecraft/class_3218;Lnet/minecraft/class_2791;Lnet/minecraft/class_2338$class_2339;D)Z
+		ARG 0 world
+		ARG 1 chunk
+		ARG 2 pos
```
Comment (liach) [inline mappings/net/minecraft/world/SpawnHelper.mapping:17]: What is ARG 3? range?

File: mappings/net/minecraft/world/biome/Biome.mapping [hunk 7]
```diff
@@ -29,13 +29,16 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 	METHOD method_24218 calculateSkyColor ()I
 	METHOD method_24374 getNoiseDistance (Lnet/minecraft/class_1959$class_4762;)F
 		ARG 1 from
+	METHOD method_24375 (Lnet/minecraft/class_1959$class_4762;Lnet/minecraft/class_1959$class_4762;)Ljava/lang/Float;
+		ARG 1 point
```
Comment (liach) [inline mappings/net/minecraft/world/biome/Biome.mapping:33]: isn't this like `first, second`?

File: mappings/net/minecraft/world/biome/layer/util/LayerSampleContext.mapping [hunk 8]
```diff
@@ -1,6 +1,8 @@
 CLASS net/minecraft/class_3628 net/minecraft/world/biome/layer/util/LayerSampleContext
 	METHOD method_15828 createSampler (Lnet/minecraft/class_4;Lnet/minecraft/class_3625;Lnet/minecraft/class_3625;)Lnet/minecraft/class_3625;
 		ARG 1 operator
+		ARG 2 sampler1
+		ARG 3 sampler2
```
Comment (liach) [inline mappings/net/minecraft/world/biome/layer/util/LayerSampleContext.mapping]:
```suggestion
		ARG 2 firstSampler
		ARG 3 secondSampler
```

File: mappings/net/minecraft/world/biome/source/MultiNoiseBiomeSource.mapping [hunk 9]
```diff
@@ -5,3 +5,5 @@ CLASS net/minecraft/class_4766 net/minecraft/world/biome/source/MultiNoiseBiomeS
 	FIELD field_22081 styleNoise Lnet/minecraft/class_3537;
 	METHOD <init> (Lnet/minecraft/class_4767;)V
 		ARG 1 config
+	METHOD method_24402 (Lnet/minecraft/class_1959$class_4762;Lnet/minecraft/class_1959;)Ljava/lang/Float;
+		ARG 1 biome
```
Comment (liach) [inline mappings/net/minecraft/world/biome/source/MultiNoiseBiomeSource.mapping:10]:
```suggestion
		ARG 0 point
		ARG 1 biome
```
Only if this method is static.

File: mappings/net/minecraft/world/biome/source/VoronoiBiomeAccessType.mapping [hunk 10]
```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_4546 net/minecraft/world/biome/source/VoronoiBiomeAccessType
 	METHOD method_22397 sqr (D)D
-		ARG 0 d
+		ARG 0 value
 	METHOD method_22398 distribute (J)D
 		ARG 0 seed
 	METHOD method_22399 calcChance (JIIIDDD)D
```
Comment (liach) [inline mappings/net/minecraft/world/biome/source/VoronoiBiomeAccessType.mapping:6]: Can try fix #1062 here

File: mappings/net/minecraft/world/chunk/PalettedContainer.mapping [hunk 11]
```diff
@@ -1,5 +1,6 @@
 CLASS net/minecraft/class_2841 net/minecraft/world/chunk/PalettedContainer
 	FIELD field_12934 paletteSize I
+	FIELD field_12935 defaultElement Ljava/lang/Object;
```
Comment (liach) [inline mappings/net/minecraft/world/chunk/PalettedContainer.mapping]:
```suggestion
	FIELD field_12935 fallback Ljava/lang/Object;
```

File: mappings/net/minecraft/world/chunk/PalettedContainer.mapping [hunk 12]
```diff
@@ -51,6 +55,7 @@ CLASS net/minecraft/class_2841 net/minecraft/world/chunk/PalettedContainer
 		ARG 2 y
 		ARG 3 z
 		ARG 4 value
+	METHOD method_19526 doesAccept (Ljava/lang/Object;)Z
```
Comment (liach) [inline mappings/net/minecraft/world/chunk/PalettedContainer.mapping]:
```suggestion
	METHOD method_19526 accepts (Ljava/lang/Object;)Z
```

File: mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping [hunk 13]
```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/class_2906 net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	FIELD field_13221 seaLevel I
 	FIELD field_13222 forcedBiome I
+	FIELD field_13223 riverSize I
+	FIELD field_13224 biomeSize I
```
Comment (liach) [inline mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping]:
```suggestion
	FIELD field_13222 forcedBiome I
```
See #1123. Insufficient evidence for these.

File: mappings/net/minecraft/world/biome/layer/util/LayerSampleContext.mapping (hunk 8 above)
Comment (jaskarth) [reply mappings/net/minecraft/world/biome/layer/util/LayerSampleContext.mapping]: there's a typo in `secodSampler`

Comment (liach) [reply mappings/net/minecraft/world/biome/layer/util/LayerSampleContext.mapping]: Done

File: mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping (hunk 13 above)
Comment (jaskarth) [reply mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping]: understood

**Comment by jaskarth** [comment]
closing as recent snapshots undid half of the work i did here.

**Pull request closed without merging**